src/app/multiple-records/multiple-records.component.spec.ts

```diff
@@ -6,6 +6,8 @@ import { SharedModule } from '@app/shared';
 import { RouterTestingModule } from '@angular/router/testing';
 import { VehicleTechRecordModel } from '@app/models/vehicle-tech-record.model';
 import { TechRecord } from '@app/models/tech-record.model';
+import { INITIAL_STATE, Store } from '@ngrx/store';
```

Comment:
We have a multiple-records.containter.ts. Hence we should not reference store in component file